MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -13,10 +13,10 @@ class RouteManeuverViewController: UIViewController {
     let routeStepFormatter = RouteStepFormatter()
     let visualInstructionFormatter = VisualInstructionFormatter()
     
-    var step: RouteStep? {
+    var currentAndUpcomingStep: (currentStep: RouteStep?, upcomingStep: RouteStep?) {
```

Comment:
A currentStep inside currentAndUpcomingStep is a bit redundant. Either leave the tuple's parts unnamed, or find a different name for the tuple itself, like displayedSteps. (One could picture us needing to add a third step to this tuple once we implement #469.